### PR TITLE
* bin/rbenv-uninstall: add kindly text into prompt

### PR DESCRIPTION
**Pull request by koooge**
```
$ rbenv uninstall 2.3.2
rbenv: remove /path/to/.rbenv/versions/2.3.2?
$
```

I don't know what I should type
and uninstallation is whether success or not.

I think this should be add kindly text like below.

```
$ rbenv uninstall 2.3.2
rbenv: remove /path/to/.rbenv/versions/2.3.2? [yN] 
Successfully uninstalled /path/to/.rbenv/versions/2.3.2
```

### CODE REVIEW

File: bin/rbenv-uninstall [hunk 1]
```diff
@@ -81,6 +81,7 @@ for hook in "${before_hooks[@]}"; do eval "$hook"; done
 if [ -d "$PREFIX" ]; then
   rm -rf "$PREFIX"
   rbenv-rehash
+  echo "Successfully uninstalled $PREFIX"
```
Comment (mislav) [inline bin/rbenv-uninstall]: Can you remove this? A Unix command usually shouldn't print that it did something successfully, unless it failed.

File: bin/rbenv-uninstall [hunk 2]
```diff
@@ -69,7 +69,7 @@ if [ -z "$FORCE" ]; then
     exit 1
   fi
 
-  read -p "rbenv: remove $PREFIX? "
+  read -p "rbenv: remove $PREFIX? [yN] "
```
Comment (mislav) [inline bin/rbenv-uninstall]: Nice addition! 👍

**Comment by koooge** [comment]
I see. I deleted success message.

$ rbenv uninstall 2.3.2
rbenv: remove /path/to/.rbenv/versions/2.3.2? [yN] y
~~Successfully uninstalled /path/to/.rbenv/versions/2.3.2~~

File: bin/rbenv-uninstall (hunk 1 above)
Comment (koooge) [inline bin/rbenv-uninstall]: You're right. 🆗

**Comment by mislav** [comment]
Thank you!

**Pull request merged**